### DISCUSSION TITLE
Use on Ubuntu Server

### DISCUSSION THREAD

**Discussion by cb-perry** [dsbenghe/Novell.Directory.Ldap.NETStandard#178] [Q&A] [answered]
Would you by chance be able to provide any guidance or details on how to get this working on Ubuntu Server? 

I currently have it successfully working for authentication on my windows machine in Visual Studio. However, when I try to get it working on my Ubuntu server it gives me the error:

"LdapException: Error stating TLA (91) Connect Error
System.Security.Authentication.AuthenticationException: The remote certificate was rejected by the provided RemoteCertificateValidationCallback.
at Novell.Directory.Ldap.AsyncExtensions.WaitAndUnwrap(Task task, Int32 timeout)
at Novell.Directory.Ldap.Connection.StartTls()"

I appreciate any help or knowledge you have! 
Thanks!

**Comment by dsbenghe** [comment, marked as answer]
My suggestion is to use some of the ldap command line utils on ubuntu i.e. ldapsearch. Get all the certificates setup correctly and after that you can try this library.

Or you can just temporarily ignore the certificates validation via provided callback and see if the library does the job for you.
